### DISCUSSION TITLE
关于useFetchs返回值

### DISCUSSION THREAD

**Discussion by MangMax** [InhiblabCore/vue-hooks-plus#192] [Q&A]
请问一下为什么useFetchs返回数据要用DeepReadonly，而useRequest只是最外面设置Readonly，这导致很多时候像赋值给相同类型的对象但是如果被加上readonly的话类型就不一样报错了，还需要自己手动处理一遍，请问一下最佳实践是怎么样子的

**Comment by NelsonYong** [comment]
resolved
